### PR TITLE
[All hosts] (publishing) Add information on creating an install link

### PR DESCRIPTION
**Pull request by davidchesnut**
This adds information on how to create an install link for your add-in. Also removes the direct link to the Microsoft AppSource publishing page and instead provides a brief overview in this content library.

### CODE REVIEW

File: docs/publish/publish-office-add-ins-to-appsource.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# Publish your Office Add-in to Microsoft AppSource
+
+Publish your Office Add-in to Microsoft AppSource to make it widely available to customers and businesses. Microsoft AppSource is an online store that contains thousands of business applications and services built by industry-leading software providers. When you publish your add-in to Microsoft AppSource, you also make it available in the in-product experience within Office.
+
+## The publishing process
+
+To include your solution in Microsoft AppSource and within Office, you submit it to Partner Center. Then it goes through an approval and certification process. For complete details, see [Make your solutions available in Microsoft AppSource and within Office](/partner-center/marketplace/submit-to-appsource-via-partner-center).
+
+## Provide an install link
```
[the remark this answers is not in the document]
Comment (lindalu-MSFT) [reply docs/publish/publish-office-add-ins-to-appsource.md:17]: Agree!
```suggestion
## Provide an installation link
```

File: docs/publish/publish-office-add-ins-to-appsource.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+After you publish to Microsoft AppSource, you can create an install link to help customers discover and install your add-in. The install link provides a "click and run" experience. Put the link on your website, social media, or anywhere you think helps your customers discover your add-in.
```
Comment (lindalu-MSFT) [inline docs/publish/publish-office-add-ins-to-appsource.md:19]:
```suggestion
After you publish to Microsoft AppSource, you can create an installation link to help customers discover and install your add-in. The installation link provides a "click and run" experience. Put the link on your website, social media, or anywhere you think helps your customers discover your add-in.
```
Do we want to say "your customers and others"?

File: docs/publish/publish-office-add-ins-to-appsource.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - For Word on the web: `2261098`
+  - For Excel on the web: `2261819`
+  - For PowerPoint on the web: `2261820`
+
+  **Note:** Outlook is not supported at this time.
+
+- **templateid**:  The ID of your add-in as listed in Microsoft AppSource.
+- **templatetitle**:  The full title of your add-in. This must be HTML encoded.
+
+For example, if you want to provide an install link for [Script Lab](https://appsource.microsoft.com/product/office/wa104380862), use the following link.
```
Comment (lindalu-MSFT) [inline docs/publish/publish-office-add-ins-to-appsource.md:40]:
```suggestion
For example, if you want to provide an installation link for [Script Lab](https://appsource.microsoft.com/product/office/wa104380862), use the following link.
```

File: docs/publish/publish-office-add-ins-to-appsource.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+[https://go.microsoft.com/fwlink/?linkid=2261819&templateid=WA104380862&templatetitle=Script%20Lab,%20a%20Microsoft%20Garage%20project](https://go.microsoft.com/fwlink/?linkid=2261819&templateid=WA104380862&templatetitle=Script%20Lab,%20a%20Microsoft%20Garage%20project)
+
+The following parameter values are used for the Script Lab install link.
```
Comment (lindalu-MSFT) [inline docs/publish/publish-office-add-ins-to-appsource.md:44]:
```suggestion
The following parameter values are used for the Script Lab installation link.
```

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**